examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,286 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name
+"""
+Graph Similarity Tutorial
+=========================
+
+This tutorial looks at how to use GBS to construct a similarity measure between graphs,
+known as a graph kernel :cite:`schuld2019quantum`. Kernels can be applied to graph-based
+data for machine learning tasks such as classification using a support vector machine.
+
+The main functionality for this tutorial is provided in :mod:`~.gbs.similarity`.
+
+Graph data
+----------
+

```

Comment:
This section is nice, but it is a bit unclear why it is right at the top here...better to move it down to the sampling section?